### PR TITLE
Execute Helm update at buildtime

### PR DESCRIPTION
**Pull request by MChorfa**
This PR introduce the instruction to update the Public Helm repos at built time , so we don't have to do it at runtime. Once the layers are cached, build time is fast and the defined repos are available

### CODE REVIEW

File: pkg/helm/build.go [hunk 1]
```diff
@@ -90,21 +91,32 @@ func (m *Mixin) Build() error {
...
-			fmt.Fprintf(m.Err, "DEBUG: addition of repository failed: %s\n", err.Error())
-		} else {
-			fmt.Fprintf(m.Out, strings.Join(commandValue, " "))
+	// Go through repositories if defined
+	if len(input.Config.Repositories) > 0 {
+		names := make([]string, 0, len(input.Config.Repositories))
+		for name := range input.Config.Repositories {
+			names = append(names, name)
+		}
+		sort.Strings(names) //sort by key we need it for the tests
```
Comment (vdice) [inline pkg/helm/build.go:100]: While I don't think sorting the repos by name is bad/wrong per se, I wonder if adding extra logic here just to satisfy testing is maybe unwarranted.  The tests can always adjust their assertions such that any order is allowed (for example, via a 'Contains' as opposed to a direct multiline match).  Not a dealbreaker by any means, but just wanted to suggest we revisit and see if this approach is worth the added code/complexity here.

In any case, if we keep the sorting, I'd suggest removing the comment about tests and updating it simply to state that we desire repeatable ordering for the repo add lines.

File: pkg/helm/build.go [hunk 2]
```diff
@@ -90,21 +91,32 @@ func (m *Mixin) Build() error {
...
+			url := input.Config.Repositories[name].URL
+			repositoryCommand, err := getRepositoryCommand(name, url)
+			if err != nil && m.Debug {
+				fmt.Fprintf(m.Err, "DEBUG: addition of repository failed: %s\n", err.Error())
+			} else {
+				fmt.Fprintf(m.Out, strings.Join(repositoryCommand, " "))
+			}
 		}
+		// Make sure we update the helm repositories
+		// So we don\'t have to do it at runtime
```
Comment (vdice) [inline pkg/helm/build.go:112]:
```suggestion
		// So we don't have to do it at runtime
```

File: pkg/helm/build.go (hunk 1 above)
Comment (MChorfa) [reply pkg/helm/build.go]: You are right, it is not a good idea to add code to just satisfy testing. So the sorting was removed. Thank you

**Review by vdice — APPROVED** [review]
LGTM, thank you @MChorfa !

**Pull request merged**